### PR TITLE
Update domains.json

### PR DESCRIPTION
**Pull request by lolieatapple**
#### Hi! What is your PR about?

- [ ] Add or edit a skin
- [X] Add a custom domain for your space
- [ ] Add an alias to migrate your space

### CODE REVIEW

**Comment by lolieatapple** [comment]
add custom vote domain for huckleberry.

File: spaces/domains.json [hunk 1]
```diff
@@ -501,5 +501,6 @@
   "dao.cronosfrens.io": "cronosfrens.io",
   "vote.tresleches.finance": "3Leches.eth",
   "gov.wagumi.xyz": "wagumi.eth",
-  "gov.tethys.finance": "tethysswap.eth"
+  "gov.tethys.finance": "tethysswap.eth",
+  "vote.huckleberry.finance":"huckleberrydex.eth"
```
Comment (gokaiorg) [inline spaces/domains.json:505]: Please add a space between sub-domain and ENS.
```suggestion
  "vote.huckleberry.finance": "huckleberrydex.eth"
```

**Review by gokaiorg — CHANGES_REQUESTED** [review]

**Comment by gokaiorg** [comment]
There is a conflict in your PR since `domains.json` file has been updated, can you resolve it?
To prevent conflicts, it is recommended to add your subdomain between two existing domains rather than at the end or beginning of the list.

**Comment by lolieatapple** [comment]
ok, I have changed it, please help to review again.

File: spaces/domains.json [hunk 2]
```diff
@@ -501,6 +501,8 @@
   "dao.cronosfrens.io": "cronosfrens.io",
   "vote.tresleches.finance": "3Leches.eth",
   "gov.wagumi.xyz": "wagumi.eth",
+  "gov.tethys.finance": "tethysswap.eth",
```
Comment (gokaiorg) [inline spaces/domains.json:504]: Please delete the duplicated row.
```suggestion
```

**Review by gokaiorg — CHANGES_REQUESTED** [review]

File: spaces/domains.json (hunk 1 above)
Comment (lolieatapple) [reply spaces/domains.json]: Ok

**Review by gokaiorg — APPROVED** [review]

**Pull request merged**